lib/tasks/import-graduation.rake

```diff
@@ -0,0 +1,56 @@
+require 'csv'
+
+files = [
+          './app/assets/data/graduation/2013-14-5YearGraduation-all.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-american_indian_or_alaskan_native.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-asian.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-asian_pacific_islander.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-black_or_african_american.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-female.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-foster_care.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-free_or_reduced_price_lunch.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-hispanic_or_latino.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-homeless.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-male.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-section_504.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-special_education.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-title_i_migrant.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-transitional_bilingual.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-two_or_more_races.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-white.csv',
+        ]
+
+namespace :db do
+  task import_graduation_data: :environment do
+    files.each do |file|
+      CSV.foreach(file, headers: true) do |row|
+        identifier_name = file.chomp('.csv').split('-')[-1].split('_').join(" ")
```

Comment:
I'm going to assume this is data you're getting from a third party. If it is, then, yeah, this is a pain point you've clearly already identified. 😞 

Mitigating this is often your only recourse unless, as @scottfirestone has suggested, you can count on your third party to produce data consistently 100% of the time. Then you can create parsers/importers that are built with a specific vendor in mind.

What you're doing here has risk, but, in my mind, is the best response. 👍